bpf/lib/maps.h

```diff
@@ -99,6 +99,16 @@ struct {
 } AUTH_MAP __section_maps_btf;
 #endif
 
+#ifdef UTIME_MAP
```

Comment:
I'm not sure the best place to document this, but I think it would be useful information for readers to see that "UTIME_MAP is a single-element array containing the boot time in UTIME format, ie nanoseconds>>9".

Reply:
Good point, adding this here:
```
/*
 * UTIME_MAP is a single-element array containing the adjusted boot time in
 * UTIME format, ie. nanoseconds>>9. "adjusted" here means that it includes the
 * time difference between BOOTTIME and MONOTONIC clocks so that we can add this
 * value to the time given by ktime_get_ns(), which does not account for suspend
 * time. Once the oldest kernel supported is Linux 5.8, we can use
 * ktime_get_boot_ns() instead so that this value does not need to contain an
 * estimate of the difference between BOOTTIME and MONOTONIC clocks any more.
 */
```

Reply:
Figured this is as good a place for this documentation as any other :-)